### PR TITLE
Fixes suggested by Sacrifice78

### PR DESCRIPTION
**Pull request by Sacrifice78**

### CODE REVIEW

File: L2J_DataPack/dist/game/data/ItemAuctions.xml [hunk 1]
```diff
@@ -30,159 +30,159 @@
 		<item auctionItemId="8" itemId="9908" itemCount="1" auctionInitBid="100000" auctionLenght="300" />
 		<!-- Life Stones -->
 		<!-- Top-Grade Life Stone - Level 80 -->
-		<item auctionItemId="9" itemId="9576" itemCount="1" auctionInitBid="100000" auctionLenght="300" />
+		<item auctionItemId="9" itemId="9576" itemCount="1" auctionInitBid="5760000" auctionLenght="300" />
```
Comment (luksdlt92) [inline L2J_DataPack/dist/game/data/ItemAuctions.xml]: That is part of the pr?

File: L2J_DataPack/dist/game/data/scripts/ai/npc/LuckyPig/LuckyPig.java [hunk 2]
```diff
@@ -0,0 +1,651 @@
...
+
+public final class LuckyPig extends AbstractNpcAI
+{
+	// Lucky Pig IDs
+	private static final int LUCKY_PIG = 2501;
+	private static final int WINGLESS_LUCKY_PIG = 2502;
+	private static final int GOLDEN_WINGLESS_LUCKY_PIG = 2503;
+	
+	// Adena Map
+	private final Map<Integer, List<Long>> adena;
```
Comment (luksdlt92) [inline L2J_DataPack/dist/game/data/scripts/ai/npc/LuckyPig/LuckyPig.java]: ```private final Map<Integer, List<Long>> _adena;```

File: L2J_DataPack/dist/game/data/scripts/ai/npc/LuckyPig/LuckyPig.java [hunk 3]
```diff
@@ -0,0 +1,651 @@
...
+	{
+		super(LuckyPig.class.getSimpleName(), "ai/npc");
+		
+		addStartNpc(LUCKY_PIG);
+		addFirstTalkId(LUCKY_PIG);
+		addTalkId(LUCKY_PIG);
+		addKillId(WINGLESS_LUCKY_PIG);
+		addKillId(GOLDEN_WINGLESS_LUCKY_PIG);
+		
+		adena = new HashMap<>();
```
Comment (luksdlt92) [inline L2J_DataPack/dist/game/data/scripts/ai/npc/LuckyPig/LuckyPig.java]: And remove this.

File: L2J_DataPack/dist/game/data/scripts/ai/npc/LuckyPig/LuckyPig.java [hunk 4]
```diff
@@ -0,0 +1,651 @@
...
+{
+	// Lucky Pig IDs
+	private static final int LUCKY_PIG = 2501;
+	private static final int WINGLESS_LUCKY_PIG = 2502;
+	private static final int GOLDEN_WINGLESS_LUCKY_PIG = 2503;
+	
+	// Adena Map
+	private final Map<Integer, List<Long>> adena;
+	
+	private static boolean isLuckyPigLevel52 = false;
```
Comment (luksdlt92) [inline L2J_DataPack/dist/game/data/scripts/ai/npc/LuckyPig/LuckyPig.java]: Could you explain me why is static? I don't remember how Lucky pig works.Whatever, _isLuckyPigLevel52.

File: L2J_DataPack/dist/game/data/scripts/ai/npc/LuckyPig/LuckyPig.java [hunk 5]
```diff
@@ -0,0 +1,651 @@
...
+		return super.onKill(npc, player, isPet);
+	}
+	
+	@Override
+	public String onSpawn(L2Npc npc)
+	{
+		switch (npc.getId())
+		{
+			case LUCKY_PIG:
+				List<Long> _Adena = new ArrayList<>();
```
Comment (luksdlt92) [inline L2J_DataPack/dist/game/data/scripts/ai/npc/LuckyPig/LuckyPig.java]: ```List<Long> adena = new ArrayList<>();```

File: L2J_DataPack/dist/game/data/scripts/events/FreyaCelebration/config.xml [hunk 6]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<event name="Freya Celebration" active="27 02 2009-28 02 2009" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../xsd/eventConfig.xsd">
+<event name="Freya Celebration" active="24 08 2010-14 09 2010" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../xsd/eventConfig.xsd">
```
Comment (luksdlt92) [inline L2J_DataPack/dist/game/data/scripts/events/FreyaCelebration/config.xml]: That is part of lucky pig? The same for the rest of events below.

File: L2J_DataPack/dist/game/data/scripts/ai/npc/LuckyPig/LuckyPig.java [hunk 7]
```diff
@@ -0,0 +1,651 @@
...
+				}
+			}
+			catch (Exception e)
+			{
+				e.printStackTrace();
+			}
+		}
+		else if (event.equals("startTalking"))
+		{
+			if (getRandomBoolean())
```
Comment (luksdlt92) [inline L2J_DataPack/dist/game/data/scripts/ai/npc/LuckyPig/LuckyPig.java]: It's not better have all the NpcStringId on an array and do

```npc.broadcastPacket(new NpcSay(npc.getObjectId(), 0, npc.getId(), strings.get(Rnd.get(strings.size - 1)));```

Or something like that?

File: L2J_DataPack/dist/game/data/ItemAuctions.xml (hunk 1 above)
Comment (Sacrifice78) [inline L2J_DataPack/dist/game/data/ItemAuctions.xml]: No, It's from Item Auction.

File: L2J_DataPack/dist/game/data/scripts/ai/npc/LuckyPig/LuckyPig.java (hunk 2 above)
Comment (Sacrifice78) [inline L2J_DataPack/dist/game/data/scripts/ai/npc/LuckyPig/LuckyPig.java]: Changed!

File: L2J_DataPack/dist/game/data/scripts/ai/npc/LuckyPig/LuckyPig.java (hunk 4 above)
Comment (Sacrifice78) [inline L2J_DataPack/dist/game/data/scripts/ai/npc/LuckyPig/LuckyPig.java]: private static boolean _isLuckyPigLevel52 = false;
private static boolean _isLuckyPigLevel70 = false;
private static boolean _isLuckyPigLevel80 = false;

File: L2J_DataPack/dist/game/data/scripts/ai/npc/LuckyPig/LuckyPig.java (hunk 7 above)
Comment (Sacrifice78) [inline L2J_DataPack/dist/game/data/scripts/ai/npc/LuckyPig/LuckyPig.java]: Not worth it ... well, if you say would be like to be ... if (blah blah blah) {} else if (blah blah blah) {} else {end bla}

File: L2J_DataPack/dist/game/data/scripts/ai/npc/LuckyPig/LuckyPig.java (hunk 5 above)
Comment (Sacrifice78) [inline L2J_DataPack/dist/game/data/scripts/ai/npc/LuckyPig/LuckyPig.java]: Changed!

File: L2J_DataPack/dist/game/data/scripts/ai/npc/LuckyPig/LuckyPig.java (hunk 3 above)
Comment (Sacrifice78) [inline L2J_DataPack/dist/game/data/scripts/ai/npc/LuckyPig/LuckyPig.java]: Removed before

File: L2J_DataPack/dist/game/data/scripts/events/FreyaCelebration/config.xml (hunk 6 above)
Comment (Sacrifice78) [inline L2J_DataPack/dist/game/data/scripts/events/FreyaCelebration/config.xml]: No, commits different

File: L2J_DataPack/dist/game/data/scripts/ai/npc/LuckyPig/LuckyPig.java (hunk 7 above)
Comment (luksdlt92) [inline L2J_DataPack/dist/game/data/scripts/ai/npc/LuckyPig/LuckyPig.java]: I don't understand :-/ I don't see necessary that if/else

File: L2J_DataPack/dist/game/data/scripts/ai/npc/LuckyPig/LuckyPig.java (hunk 4 above)
Comment (luksdlt92) [inline L2J_DataPack/dist/game/data/scripts/ai/npc/LuckyPig/LuckyPig.java]: And why is static? :dancer:

File: L2J_DataPack/dist/game/data/scripts/events/FreyaCelebration/config.xml (hunk 6 above)
Comment (luksdlt92) [inline L2J_DataPack/dist/game/data/scripts/events/FreyaCelebration/config.xml]: Ok, remove it from the commit and the rest of unnecessary files (git rm dir/file)

**Comment by Sacrifice78** [comment]
Close the pull request. I do not want to bother... we can see on forum.

**Pull request closed without merging**